nix/autofirma/patches/clienteafirma/etc_config.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/afirma-simple/src/main/java/es/gob/afirma/standalone/protocol/SecureSocketUtils.java b/afirma-simple/src/main/java/es/gob/afirma/standalone/protocol/SecureSocketUtils.java
+index 4485c27aa..6e96cb830 100644
+--- a/afirma-simple/src/main/java/es/gob/afirma/standalone/protocol/SecureSocketUtils.java
++++ b/afirma-simple/src/main/java/es/gob/afirma/standalone/protocol/SecureSocketUtils.java
+@@ -65,7 +65,8 @@ class SecureSocketUtils {
+ 	 * @return Almac&eacute;n de claves o {@code null} si no se encontr&oacute;. */
+ 	private static File getKeyStoreFile() {
+ 
+-		File appDir = AutoFirmaUtil.getApplicationDirectory();
++		// Primera opción: /etc/AutoFirma en lugar de getApplicationDirectory()
```

Comment:
Since this patch’s purpose is to be applied while building on a nix system, and not to be upstreamed, I’d suggest using only English for the comments, in order not to alienate our non-Spanish speaking users.